### PR TITLE
Feature/fix metastore logging

### PR DESCRIPTION
**Pull request by rpoluri**

### CODE REVIEW

File: files/hive-log4j2.properties [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (massdosage) [inline files/hive-log4j2.properties]: Is this based on some example/default from Hive itself? If so we should reference it. I'm assuming they don't have an XML equivalent?

File: files/startup.sh [hunk 2]
```diff
@@ -75,6 +75,7 @@ fi
 fi
 
 [[ -z $HIVE_METASTORE_LOG_LEVEL ]] && HIVE_METASTORE_LOG_LEVEL="INFO"
```
Comment (pradeepbhadani) [inline files/startup.sh]: Why ` && HIVE_METASTORE_LOG_LEVEL="INFO"` ?

Comment (rpoluri) [reply files/startup.sh]: This sets default vault if HIVE_METASTORE_LOG_LEVEL variable is not passed to Docker container.

File: files/hive-log4j2.properties (hunk 1 above)
Comment (rpoluri) [reply files/hive-log4j2.properties]: This file is installed with EMR hive, migh't have originated from apache hive, will try to find source and reference. Yeah didn't find XML equivalent.

File: files/startup.sh (hunk 2 above)
Comment (pradeepbhadani) [reply files/startup.sh]: okay.

**Pull request merged**